cub/cub/device/dispatch/dispatch_transform.cuh

```diff
@@ -346,15 +391,16 @@ struct dispatch_t<StableAddress,
   CUB_RUNTIME_FUNCTION _CCCL_FORCEINLINE cudaError_t Invoke(ActivePolicyT active_policy = {})
   {
     auto wrapped_policy = detail::transform::MakeTransformPolicyWrapper(active_policy);
+    auto seq            = ::cuda::std::index_sequence_for<RandomAccessIteratorsIn...>{};
 #ifdef _CUB_HAS_TRANSFORM_UBLKCP
     if constexpr (Algorithm::ublkcp == wrapped_policy.GetAlgorithm())
```

Comment:
Question: Does this need the python treatment?

Reply:
Yes, but that's done in #2394.